src/model/MultiValueColumn.ts

```diff
@@ -86,151 +89,130 @@ export default class MultiValueColumn extends ValueColumn<number[] > {
 
   constructor(id: string, desc: any) {
     super(id, desc);
-    this.colorrange = (<any>desc).colorrange || ['blue', 'red'];
+    this.colorRange = (<any>desc).colorRange || ['blue', 'red'];
     this.min = d3.min((<any>desc).domain);
     this.max = d3.max((<any>desc).domain);
-    this.bins = (<any>desc).datalength;
+    this.dataLength = (<any>desc).dataLength;
     this.threshold = (<any>desc).threshold || 0;
-
     this.sortBy = (<any>desc.sort) || 'min';
-    this.verticalBarHeight = 13;
-    this.rendererList = [{type: 'heatmapcustom', label: 'Heatmap'},
+    this.cellWidth = this.getWidth() || 100;
+    this.rowHeight = 13;
+    this.dataInfo = {
```

Comment:
why is the information `min, max, threshold, sort` stored twice. e.g, once in `this.sortBy` and `this.dataInfo.sortBy`. Which one is the right one, which one is the one to update?